src/components/molecules/definitionitem/DefinitionItem.scss

```diff
@@ -30,7 +34,7 @@ a.dictionary-definiton__patient-information-button {
 	align-items: center;
 	font-family: Montserrat, Avant Garde, Arial, sans-serif;
 	font-size: 16px;
-	line-height: normal;
+	line-height: 29px;
```

Comment:
Line heights should be unitless wherever possible.  1.75 looks about right here.